INSTALL

```diff
@@ -189,7 +189,7 @@ GitHub, you have to perform a few steps. In general:
 
 In detail:
 
-    cd /where/your/grass7sourcecode/lives/
+    cd /where/your/grass8sourcecode/lives/
```

Comment:
This should definitively say `grass-source-code` without the number.

Reply:
fixed in 5bd452e4e